apps/web/src/pages/Pools/PoolAdvertisementPage/components/GenericJobTitleAccordion.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { Accordion, StandardAccordionHeader } from "@gc-digital-talent/ui";
+
+import { GenericJobTitle } from "~/api/generated";
+import { getLocalizedName } from "@gc-digital-talent/i18n";
+import ClassificationDefinition from "./ClassificationDefinition";
+
+interface GenericJobTitleAccordionProps {
+  genericJobTitle: GenericJobTitle;
+  suffix: string;
```

Comment:
I think this is a prefix? (comes before the title name, yes?)

Reply:
heh, not sure what I was thinking there 😅 Fixed in [update generic job title props to reflect usage](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6543/commits/c6427d54c2f131a1a07de115854cfd3c03b48479)

Reply:
cool thx!